types/event_bus.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/cometbft/cometbft/libs/service"
 )
 
-const defaultCapacity = 0
+const defaultCapacity = 10000
```

Comment:
oooof

Reply:
Commit: aaddf5d32 by Anton Kaliaev
  Reason: Bug detection and debugging
  Context from commit message:
  - Referenced issue https://github.com/tendermint/tendermint/issues/951
  - Quote from Jae: "forcing pubsub's internal channel to have a capacity of 0 will reveal bugs sooner, if the subscriber also has a 0 or small capacity ch to pull from"
  - Purpose: Zero capacity forces synchronous behavior, making concurrency bugs more apparent during testing and development

Reply:
Hmm well if there's a problem with slow reads even the 10000 buffer will eventually get filled and slow down consensus.

Perhaps you want a mechanism to drop the subscription after it hits a certain limit. 

If the e2e test has the indexer disabled does it pass? 

I don't mind trying out this fix. I actually think it's still quite bad if bridge nodes drop behind consensus because that's what rollup sequencers depend on